kotlin-ir-plugin/src/test/kotlin/com/bnorm/template/IrPluginTest.kt

```diff
@@ -42,19 +46,18 @@ fun debug() = "Hello, World!"
 
 fun compile(
   sourceFiles: List<SourceFile>,
-  plugin: ComponentRegistrar = TemplateComponentRegistrar(),
-): KotlinCompilation.Result {
+  plugin: CompilerPluginRegistrar = TemplateCompilerRegistrar(),
```

Comment:
I'm not sure what ktlint is complaining about here... The only thing I can think of is you put the `@file:OptIn(ExperimentalCompilerApi::class)` before the copyright block?

Reply:
I'll try fixing that when I'm back on my personal computer.

Reply:
I think this may actually be because of a really old GitHub action version for KtLint. I'm going to merge this PR anyways since I can investigate that separately.